alacritty/src/event.rs

```diff
@@ -375,7 +388,7 @@ impl ApplicationHandler<Event> for Processor {
                 self.scheduler.unschedule_window(window_context.id());
 
                 // Shutdown if no more terminals are open.
-                if self.windows.is_empty() {
+                if self.windows.is_empty() && !self.cli_options.daemon {
                     // Write ref tests of last window to disk.
                     if self.config.debug.ref_test {
```

Comment:
I think we should disable `ref-test` with `daemon` mode in `cli.rs` all together. Though maybe remove `ref-test` all together from release builds.

Reply:
I'm in favor of removing `ref-test` from release if it easy to do so.

Reply:
I am very much against removing `ref-test` from release builds.

Reply:
Could just make it hidden, I guess. Though, it's all irrelevant for this PR. The issue here is the ref-test shouldn't work for daemon mode, otherwise it'll endlessly collect stuff IIRC.

Reply:
I've made no changes here because `ref_test` can be set from both CLI and config, so I don't see any reason for adding additional complexity for half-measures here.